### ISSUE TITLE
MOV from iPhone is unseekable

### ISSUE THREAD

**Issue by PacoH** [mpv-player/mpv#13013]
MOV files from iPhone are unseekable

- mpv version: `mpv 0.37.0`
	- built on Nov 22 2023 02:28:54
- macOS Version: macOS Ventura 13.3.1
- Source: `brew install mpv`
- Reproducible with `--no-config`
- Expected behavior:
	- Should be seekable as with any other codec
- Actual behavior:
	- Not seekable
		- Any attempt to seek starts the video over again
- Log files:
	- [output\.txt](https://0x0.st/Hx1d.txt)
- Sample files:
	- [MOV file](https://0x0.st/Hx1v.MOV)

**Comment by orion1vi** [comment]
https://github.com/mpv-player/mpv/issues/12047

**Comment by Akemi** [comment]
duplicate of https://github.com/mpv-player/mpv/issues/12047

**Comment by PacoH** [comment]
I don't think this is a duplicate. In issue linked to, seeking takes place but is inaccurate. In my case seeking fails completely.

>Actual behavior
Playback restarts at an earlier time.
Running the test script shows the time-pos property initially advancing by 5 seconds as expected and one second later time-pos reports the playback position is earlier than when the seek command was submitted

I tried using `--hr-seek` plus `--no-config` and this works just fine. But with `--no-config` alone I get the behavior that I reported.

So the problem seems to exist in `mpv`. At least with MOVs from iPhone. This didn't used to happen before, so maybe there is some setting that changed on the iPhone Camera app.

Mediainfo:
- Format profile: QuickTime
- Codec ID: hvc1
